languages/csharp/concepts/attributes/links.json

```
@@ -1 +1,22 @@
-[]
+[
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/attributes/",
+    "description": "attribute concept"
+  },
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/attributes",
+    "description": "attribute tutorial"
+  },
+  {
+    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/attributes/#attribute-targets",
+    "description": "attribute target"
+  },
```

I would remove the attribute targets link (as it is contained as part of the first link's page) and update the text a bit:

```suggestion
  {
    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/attributes/",
    "description": "Attributes"
  },
  {
    "url": "https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/attributes",
    "description": "Tutorial"
  },
```

This is how the links will look like BTW: https://github.com/exercism/v3/issues/2499